### PR TITLE
you can now bite into an aggressively grabbed seelie and they explode like a fucking gusher

### PR DESCRIPTION
**Pull request by Slorbin**
## About The Pull Request

https://github.com/user-attachments/assets/83bf1013-8b3e-4606-a4b4-465e7f028d80

this body horror enough for you?

## Why It's Good For The Game

much funnier than before

### CODE REVIEW

**Comment by Gyrandola** [comment]
okay make it less OP

**Comment by DeerJesus** [comment]
]

> okay make it less OP

updated to fit the requests made in discord:

Only graggarites can do it

Requires a do_after of 5 seconds

File: code/_onclick/other_mobs.dm [hunk 1]
```diff
@@ -205,6 +205,15 @@
...
 		if(!lying_attack_check(user))
 			return FALSE
+	
+	if(isseelie(src))
+		if(user.patron.type == /datum/patron/inhumen/graggar)
+			if(user.pulling == src)
+				if(user.grab_state == GRAB_AGGRESSIVE)
+					visible_message(span_danger("[user] is putting [src] in their mouth!"), \
+									span_userdanger("[user] is putting me in their mouth!"))
+					if(do_after(user, 5 SECONDS, target = src))
```
Comment (TheFenix323) [inline code/_onclick/other_mobs.dm:215]: If anything make it take as long as wing ripping, if not more since gibbing them would make them unrevivable

Comment (DeerJesus) [reply code/_onclick/other_mobs.dm:215]: how long is ring wipping

Comment (TheFenix323) [reply code/_onclick/other_mobs.dm:215]: image

**Comment by Crowbar764** [comment]
Someone get the seelies away from this man

**Comment by Kevak** [comment]
Finally! You stopped being a filthy coward! This is much better! +1

**Comment by RyszardParys** [comment]
Why..?

**Comment by DeerJesus** [comment]
Proved to be a lot funnier in theory than in practice

**Pull request closed without merging**